packages/web/components/Modal/CurateTechnologyModal/__tests__/index.js

```diff
@@ -24,40 +36,53 @@ describe('<CurateTechnologyModal />', () => {
 
 		render(<CurateTechnologyModal technology={fakeTechnology} closeModal={closeModal} />);
 
+		await screen.findByText(fakeTechnology.title);
+
 		tabsName.forEach((tab) => {
 			const tabElement = screen.getByRole('tab', { name: tab });
-			fireEvent.click(tabElement);
+
 			expect(tabElement).toBeInTheDocument();
 		});
 	});
 
-	it('should close modal when clicking X button', () => {
+	it('should close modal when clicking X button', async () => {
 		render(<CurateTechnologyModal technology={fakeTechnology} closeModal={closeModal} />);
 
-		fireEvent.click(screen.getByLabelText(/close modal/i));
+		await screen.findByText(/Agebavzi niko zaro\./i);
```

Comment:
Please change `/Agebavzi niko zaro\./i` to something more significant like `technology.title` (in all these occurrences) 😅